test/tests/functional/pbs_pbsnodes_attributes.py

```diff
@@ -0,0 +1,115 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class Testpbsnodes_attributes(TestFunctional):
+
+    def Test_pbsnodes(self, user):
+        """
+        Function to validate pbsnodes attribute on user basis
```

Comment:
I'm not sure I understand what this function is doing, is it validating that the default attributes that are set on a vnode are set correctly?

Reply:
validating values of pbsnodes attributes are correct or not.

Reply:
pbsnodes is a command, so I'm guessing you are talking about node attributes, and since you are not setting any explicitly in the test, it seems like it is testing whether the default values of node attributes are set correctly or not. So, I suggest changing the docstring to say that.